docs/index.md

```diff
@@ -1,26 +1,28 @@
-# Welcome to the EasyBuild documentation!
+# EasyBuild documentation
 
-!!! warning
-    This is the new EasyBuild documentation,
-    made with [Material for MkDocs](https://squidfunk.github.io/mkdocs-material)
-    and written in [MarkDown](https://daringfireball.net/projects/markdown).
+![EasyBuild logo](img/easybuild_logo_2022_vertical_dark_bg_transparent.png){: align=left style="height:455px;width:400px"}
 
-    We intend to replace the <https://docs.easybuild.io>,
-    which is made with [Sphinx](https://www.sphinx-doc.org)
-    and written in [reStructuredText](https://en.wikipedia.org/wiki/ReStructuredText)
-    with this documentation soon...
+Welcome to the documentation of **[EasyBuild](https://easybuild.io) (version {{ easybuild.version }})**.
 
-    **
-    If you want to help with porting the EasyBuild documentation to MarkDown, you can!  
-    Please see <https://github.com/easybuilders/easybuild-docs>.
-    **
+Use the menu to navigate, or use the search box (top right).
 
-## Getting started
+See also the [EasyBuild tutorial](https://easybuilders.github.io/easybuild-tutorial) (separate website).
 
-- [What is EasyBuild?](what-is-easybuild.md)
-- [EasyBuild terminology](terminology.md)
-- [Installing](installation.md) and [configuring](configuration.md) EasyBuild
-- [Using EasyBuild](using-easybuild.md)
-- [Getting help](getting-help.md)
+To contribute to this documentation, [open a pull request](https://github.com/easybuilders/easybuild-docs).
 
-## Introductory topics {: #contents }
```

Comment:
```
mkdocs_autorefs.plugin: typical-workflow-example-with-wrf.md: Could not find cross-reference target '[contents]'
```

So that link in `typical-workflow-example-with-wrf.md` should just point to the homepage?

Reply:
I'll point to `using-easybuild.md` instead